File: Python/1_rev_arr.py
def rev_ls(A, start, end):
	while start < end:
		A[start], A[end] = A[end], A[start]
		start += 1
		end -=1

A = [1, 2, 3, 4, 5, 92, 87]
print (A)
rev_ls(A, 0, 6)
print(A)

File: C/1_rev_arr.c
#include <stdio.h>

void pr_arr(int arr[], int size);
void rev_arr(int arr[], int start, int end);

int main()
{
	int arr[] = {1, 2, 3, 4, 5, 29, 32, 99, 884};	
	int n = sizeof(arr) / sizeof(arr[0]);
	pr_arr(arr, n);
	rev_arr(arr, 0, n-1);
	pr_arr(arr, n);
	return 0;
}

void pr_arr(int arr[], int size)
{
	for (int i = 0; i < size; i++)
		printf("%d ", arr[i]);
	printf("\n");
}

void rev_arr(int arr[], int start, int end)
{
	while (start < end) {
		int temp = arr[start];
		arr[start] = arr[end];
		arr[end] = temp;
		start++;
		end--;
	}
}
